src/pages/settings/Wallet/ChooseTransferAccountPage.tsx

```diff
@@ -50,40 +51,73 @@ function ChooseTransferAccountPage({walletTransfer = {}}: ChooseTransferAccountP
         BankAccounts.openPersonalBankAccountSetupView();
     };
 
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const selectedAccountID = walletTransfer?.selectedAccountID;
+    const data = useMemo(() => {
+        const options = Object.values(bankAccountsList ?? {}).map((bankAccount) => {
+            const bankName = (bankAccount.accountData?.additionalData?.bankName ?? '') as BankName;
+            const bankAccountNumber = bankAccount.accountData?.accountNumber ?? '';
+            const bankAccountID = bankAccount.accountData?.bankAccountID ?? bankAccount.methodID;
+            const {icon, iconSize, iconStyles} = getBankIcon({bankName, styles});
+            return {
+                value: bankAccountID,
+                text: bankAccount.title,
+                leftElement: icon ? (
+                    <View style={[styles.flexRow, styles.alignItemsCenter, styles.mr3]}>
+                        <Icon
+                            src={icon}
+                            width={iconSize}
+                            height={iconSize}
+                            additionalStyles={iconStyles}
+                        />
+                    </View>
+                ) : null,
+                alternateText: `${translate('workspace.expensifyCard.accountEndingIn')} ${getLastFourDigits(bankAccountNumber)}`,
+                keyForList: bankAccountID?.toString(),
+                isSelected: bankAccountID?.toString() === selectedAccountID,
+                bankAccount,
+            };
+        });
+        return options;
+    }, [bankAccountsList, selectedAccountID, styles, translate]);
+
+    if (isLoadingOnyxValue(walletTransferResult)) {
+        return <FullscreenLoadingIndicator />;
+    }
+
     return (
         <ScreenWrapper testID={ChooseTransferAccountPage.displayName}>
             <HeaderWithBackButton
                 title={translate('chooseTransferAccountPage.chooseAccount')}
                 onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WALLET_TRANSFER_BALANCE)}
             />
-            <View style={[styles.mt3, styles.flexShrink1, styles.flexBasisAuto]}>
-                <PaymentMethodList
-                    onPress={selectAccountAndNavigateBack}
-                    shouldShowSelectedState
-                    filterType={walletTransfer?.filterPaymentMethodType}
-                    selectedMethodID={walletTransfer?.selectedAccountID}
-                    shouldShowAddPaymentMethodButton={false}
-                    shouldShowAddBankAccount={false}
-                    shouldShowRightIcon={false}
+            <ScrollView>
```

Comment:
Why do we need to add `ScrollView` wrapper?

Reply:
@allgandalf sorry I haven't t posted it earlier, but I've just reminded that I had a very similar problem with SelectionList while working on TransactionStartDate step (for assigning card flow).

I used this styles for SelectionList to have this menuItem right after the list
<img width="330" alt="Screenshot 2024-09-19 at 13 35 44" src="https://github.com/user-attachments/assets/a12819cb-4470-4595-841b-bb70bf63c359">

`containerStyle={[styles.flex0, styles.flexShrink0, styles.flexBasisAuto, styles.pb0]}`

https://github.com/koko57/App/blob/034349a94361480c958a09f9a86d8163e4fcd4d0/src/pages/workspace/companyCards/assignCard/TransactionStartDateStep.tsx#L102

(I see that somehow the commented styles were'nt watched during the review, I will fix this in my other PR 😂 )

Reply:
FYI: styles.flexReset doesn't work on mobile here, so that's why I styled it this way